core/domain/exp_services.py

```diff
@@ -554,6 +555,117 @@ def apply_change_list(exploration_id, change_list):
                         'version %s, received %s' % (
                             feconf.CURRENT_STATE_SCHEMA_VERSION,
                             change.to_version))
+
+        # Calculate the size of exp.
+        state_protos = {}
```

Comment:
@BenHenning Could you please take a look at this file and check if we are going on the correct path in parsing the Protos?